jellyfin_kodi/jellyfin/connection_manager.py

```diff
@@ -243,18 +217,17 @@ def _connect_to_servers(self, servers, options):
         }
 
     def _try_connect(self, url, timeout=None, options={}):
-
         url = self.get_jellyfin_url(url, "system/info/public")
         LOG.info("tryConnect url: %s", url)
+        timeout = timeout or self.timeout
+        headers = self._get_headers()
 
-        return self._request_url({
-            'type': "GET",
-            'url': url,
-            'dataType': "json",
-            'timeout': timeout,
-            'verify': options.get('ssl'),
-            'retry': False
-        })
+        try:
+            return self.http.request_url(url, "GET", headers=headers, \
+                    timeout=timeout, retry=False)
```

Comment:
Newline is allowed between function parameters, there is no need to escape the newline.